packages/data-context/src/sources/BrowserDataSource.ts

```diff
@@ -1,15 +1,19 @@
 import type { FoundBrowser, BrowserStatus } from '@packages/types'
 import os from 'os'
-import { execSync } from 'child_process'
+import execa from 'execa'
+
 import type { DataContext } from '..'
 
 let isPowerShellAvailable = false
 
-try {
-  execSync(`[void] ''`, { shell: 'powershell' })
-  isPowerShellAvailable = true
-} catch {
-  // Powershell is unavailable
+// Only need to worry about checking for PowerShell in windows,
+// doing it asynchronously so to not block startup
+if (os.platform() === 'win32') {
+  execa(`[void] ''`, { shell: 'powershell' }).then(() => {
+    isPowerShellAvailable = true
+  }).catch(() => {
+    // Powershell is unavailable
+  })
```

Comment:
@ryanthemanuel just wanted to confirm this change was ok / there wasn't a reason this needed to be done synchronously during script loading?

Reply:
I believe this should be ok. I think the weird scenario would be if this takes longer to resolve than when we get to the OpenBrowserList page:

https://github.com/cypress-io/cypress/blob/10.0-release/packages/launchpad/src/setup/OpenBrowserList.vue#L113

i guess we could guard around that by ensuring to wait for the result here:

https://github.com/cypress-io/cypress/pull/20997/files#diff-ccfea29f817d69316d07e1e2ad570f26a2125e956e357df65244757a12331cd3R75